packages/app/src/runner/index.ts

```diff
@@ -115,12 +114,16 @@ function createIframeModel () {
  * for communication between driver, runner, reporter via event bus,
  * and server (via web socket).
  */
-function setupRunner (namespace) {
+function setupRunner (namespace: AutomationElementId) {
   const mobxRunnerStore = getMobxRunnerStore()
+  const runnerUiStore = useRunnerUiStore()
 
   getEventManager().addGlobalListeners(mobxRunnerStore, {
+    randomString: runnerUiStore.randomString,
+    setAutomationStatus: (status: AutomationStatus) => {
+      runnerUiStore.setAutomationStatus(status)
+    },
```

Comment:
Do we need this prop? not sure if we are actually using it

Reply:
nope